pkgs/development/coq-modules/autosubst/default.nix

```diff
@@ -1,15 +1,18 @@
-{ lib, mkCoqDerivation, coq, mathcomp-ssreflect, version ? null }:
+{ lib, mkCoqDerivation, coq, mathcomp-ssreflect,
+  version ? null, origin ? null }:
 with lib;
 
 mkCoqDerivation {
-  pname = "autosubst";
+  pname   = "autosubst";
+  owner   = "uds-psl";
+  inherit version origin;
 
   release."1.7".rev    = "v1.7";
   release."1.7".sha256 = "sha256-qoyteQ5W2Noxf12uACOVeHhPLvgmTzrvEo6Ts+FKTGI=";
 
-  inherit version;
   defaultVersion = with versions; switch coq.coq-version [
-    { case = isGe "8.10"; out = "1.7"; }
+    { case = isGe "8.10";       out = "1.7"; }
+    { case = range "8.5" "8.7"; out = "5b40a32e"; }
```

Comment:
This is a commit outside the main repository. Are you sure about this?

Reply:
no